SoftLayer/CLI/virt/usage.py

```diff
@@ -0,0 +1,53 @@
+"""Usage information of a virtual server."""
+# :license: MIT, see LICENSE for more details.
+
+import click
+
+import SoftLayer
+from SoftLayer.CLI import environment
+from SoftLayer.CLI import formatting
+from SoftLayer.CLI import helpers
+from SoftLayer.utils import clean_time
+
+
+@click.command()
+@click.argument('identifier')
+@click.option('--start_date', '-s', type=click.STRING, required=True, help="Start Date e.g. 2019-3-4 (yyyy-MM-dd)")
+@click.option('--end_date', '-e', type=click.STRING, required=True, help="End Date e.g. 2019-4-2 (yyyy-MM-dd)")
+@click.option('--valid_type', '-t', type=click.STRING, required=True,
+              help="Metric_Data_Type keyName e.g. CPU0, CPU1, MEMORY_USAGE, etc.")
+@click.option('--summary_period', '-p', type=click.INT, default=1800,
+              help="300, 600, 1800, 3600, 43200 or 86400 seconds")
+@environment.pass_env
+def cli(env, identifier, start_date, end_date, valid_type, summary_period):
+    """Usage information of a virtual server."""
+
+    vsi = SoftLayer.VSManager(env.client)
+    table = formatting.Table(['counter', 'dateTime', 'type'])
+    table_average = formatting.Table(['Average'])
+
+    vs_id = helpers.resolve_id(vsi.resolve_ids, identifier, 'VS')
+
+    result = vsi.get_summary_data_usage(vs_id, start_date=start_date, end_date=end_date,
```

Comment:
add a check to make sure we actually get results back, which will avoid a divide by 0 error.

```
$ ./slcli -vvv  vs usage -s 2019-01-01 -e 2019-02-01  -t CPU0 -p 600  76729221
Calling: SoftLayer_Virtual_Guest::getMetricTrackingObjectId(id=76729221, mask='', filter='None', args=(), limit=None, offset=None))

https://api.softlayer.com:443 "GET /rest/v3.1/SoftLayer_Virtual_Guest/76729221/getMetricTrackingObjectId.json 

Returned Data:
41768785

Calling: SoftLayer_Metric_Tracking_Object::getSummaryData(id=41768785, mask='', filter='None', args=('2019-01-01', '2019-02-01', [{'keyName': 'CPU0', 'summaryType': 'max'}], 600), limit=100, offset=0))

https://api.softlayer.com:443 "POST /rest/v3.1/SoftLayer_Metric_Tracking_Object/41768785/getSummaryData.json?resultLimit=0%2C100 HTTP/1.1" 

Returned Data:
[]

An unexpected error has occured:
Traceback (most recent call last):
  File "C:\Users\allmi\Source\softlayer-python\SoftLayer\CLI\core.py", line 182, in main
    cli.main(**kwargs)
  File "C:\Users\allmi\Source\py36\lib\site-packages\click\core.py", line 717, in main
    rv = self.invoke(ctx)
  File "C:\Users\allmi\Source\py36\lib\site-packages\click\core.py", line 1137, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "C:\Users\allmi\Source\py36\lib\site-packages\click\core.py", line 1137, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "C:\Users\allmi\Source\py36\lib\site-packages\click\core.py", line 956, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "C:\Users\allmi\Source\py36\lib\site-packages\click\core.py", line 555, in invoke
    return callback(*args, **kwargs)
  File "C:\Users\allmi\Source\py36\lib\site-packages\click\decorators.py", line 64, in new_func
    return ctx.invoke(f, obj, *args, **kwargs)
  File "C:\Users\allmi\Source\py36\lib\site-packages\click\core.py", line 555, in invoke
    return callback(*args, **kwargs)
  File "C:\Users\allmi\Source\softlayer-python\SoftLayer\CLI\virt\usage.py", line 48, in cli
    average = counter / count
ZeroDivisionError: float division by zero

Feel free to report this error as it is likely a bug:
    https://github.com/softlayer/softlayer-python/issues
The following snippet should be able to reproduce the error
(py36)
```

Reply:
Done.